opentech/apply/funds/models.py

```diff
@@ -74,6 +77,18 @@ def next_deadline(self):
     content_panels = AbstractStreamForm.content_panels + [
         FieldPanel('workflow'),
         InlinePanel('forms', label="Forms"),
+        MultiFieldPanel(
+            [
+                FieldRowPanel([
+                    FieldPanel('from_address', classname="col6"),
+                    FieldPanel('to_address', classname="col6"),
+                ]),
+                FieldPanel('subject'),
+                FieldPanel('confirmation_text_extra'),
+            ],
+            heading="Confirmation email",
+            classname="collapsible collapsed"
+        ),
```

Comment:
Maybe a new tab for this? "email settings" 

Not essential, but my preference.

http://docs.wagtail.io/en/v1.13.1/advanced_topics/customisation/page_editing_interface.html#customising-the-tabbed-interface